src/client/components/app-core/form.css

```diff
@@ -0,0 +1,33 @@
+label {
+  cursor: pointer;
```

Comment:
I think this will come and bite you, but I can see it's value

Reply:
After discussion, I agree and removed the global styling.